flexinput/src/main/java/com/lytefast/flexinput/adapters/PhotoCursorAdapter.kt

```diff
@@ -242,5 +241,11 @@ class PhotoCursorAdapter(private val contentResolver: ContentResolver,
         TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, dipValue, resources.displayMetrics)
 
     fun isAndroidQ() = Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q
+
+    fun Context.themeColor(@AttrRes attrRes: Int): Int {
+      val typedValue = TypedValue()
+      theme.resolveAttribute(attrRes, typedValue, true)
```

Comment:
I thought you needed contextcompat was because theme could be null?

Reply:
Good call